src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -302,23 +313,17 @@ export const WorkspaceDashboard = _.flow(
           'Google Bucket'
         ]),
         div({ style: { display: 'flex' } }, [
-          div({ style: Style.noWrapEllipsis }, [bucketName]),
+          (bucketLocation ?
```

Comment:
Thinking about the presentation of information here: We've already established a section that's about the bucket, so we don't necessarily need to squeeze everything onto one line. What about leaving this one as-is, and adding another line beneath with the region info? That way we could also display the region text directly, without relying on tooltips.

Reply:
Thanks - I've restored the original link text and split the content across two lines:

![image](https://user-images.githubusercontent.com/13006282/101829935-cae5dc80-3b01-11eb-8766-68838af899e3.png)